css-fonts-4/Overview.bs

```diff
@@ -7058,6 +7064,25 @@ Animation type: discrete
 	    be a modification of an existing palette already present in the font,
 	    or be an entirely new palette.
 
+	<dt><dfn type><<palette-mix()>></dfn>
+	<dd>
+        This value defines a position in the interpolation between two palette values,
+        identified by <<palette-identifier>>, by the palette keywords
+        ''font-palette/light'' or ''font-palette/dark'',
+        or by another <<palette-mix()>> value.
+
+        The glyphs of the font must be rendered with a palette that is created by
+        interpolating for each palette color index between the matching color indices
+        of the first and second palette specified as arguments,
+        recursively resolving <<palette-mix()>> functions if needed.
+
+        For each palette color, the interpolation follows
```

Comment:
Does this just mean that it uses the [&lt;color-interpolation-method>](https://drafts.csswg.org/css-color-4/#color-interpolation-method) token and in general follows CSS Color 4 [Color Interpolation](https://drafts.csswg.org/css-color-4/#interpolation) or is there a tighter dependency on `color-mix()`, for example normalization of percentage values?

Reply:
Yes, as you noticed, percentage normalization specified  below.